tasks/deploy-widgets.js

```diff
@@ -5,64 +5,90 @@ import path from 'path';
 import request from 'request';
 import evrythng from 'evrythng-extended';
 
-import webpackConfig from '../webpack.config';
-import getApiKey from './get-api-key';
-import projectConfig from './config';
+import webpackDistConfig from '../webpack.dist.config';
+import config from './config';
 
-const config = {
+const taskConfig = {
   file: path.join(
-    webpackConfig.output.path,
-    webpackConfig.output.filename.replace('[name]', projectConfig.name)
+    webpackDistConfig.output.path,
+    webpackDistConfig.output.filename.replace('[name]', config.name)
   ),
   filesFilter: {
     tags: ['components']
   }
 };
 
-////////////////////////////////////////////
-
+/**
+ * Deploys the built components to EVRYTHNG Files API.
+ *
+ * @return {Promise}
+ */
 export default () => {
-  return getApiKey()
+  return readAPIKeyFromEnvironment()
```

Comment:
environment means environment variable? no need to add implementation details to the method name (we can change the way you read it at some point in the future). simply `readApiKey` is fine